src/crate/theme/rtd/crate/static/css/crateio-rtd.css

```diff
@@ -40,24 +40,13 @@ blockquote {
 
 /** admonitions **/
 
-div.admonition p {
```

Comment:
removing the paragraph and ul styles gives this result (above is the original, below the changes):
I would think that the regular paragraphs should be below the headline, no? What do you think @mechanomi 
![grafik](https://user-images.githubusercontent.com/8767096/109145454-c9bc6700-7762-11eb-8f43-87d0bf0cd7bb.png)